Assets/Plugins/Source/EOSManager.cs

```diff
@@ -453,13 +453,36 @@ public void Init(IEOSCoroutineOwner coroutineOwner, string configFileName)
 
                 //TODO: provide different way to load the config file?
                 string eosFinalConfigPath = System.IO.Path.Combine(Application.streamingAssetsPath, "EOS", configFileName);
+                
+                var configDataAsString = "";
+
+                string configDataAsString = "";
+
+#if UNITY_ANDROID
+                using (var request = UnityEngine.Networking.UnityWebRequest.Get(eosFinalConfigPath))
+                {
+                    request.timeout = 2; //seconds till timeout
+                    request.SendWebRequest();
 
+                    //Wait till webRequest completed
+                    while (!request.isDone) { }
+
+                    if (request.result != UnityEngine.Networking.UnityWebRequest.Result.Success)
+                    {
+                        UnityEngine.Debug.LogWarning("Requesting " + eosFinalConfigPath + ", please make sure it exists and is a valid config");
```

Comment:
These calls to Unity's logging need to be changed to calls to 'print' so that we can allow users of the plugin to turn off all the logging with `ENABLE_DEBUG_EOSMANAGER`

Reply:
Completed